packages/components/src/local/organisms/planning-channel/index.tsx

```diff
@@ -711,9 +711,10 @@ export const PlanningChannel: React.FC<PropTypes> = ({
   useEffect(() => {
     // drop the turn markers
     const nonTurnMessages: Array<MessagePlanning | MessageInteraction> = messages.filter((msg: MessagePlanning | MessageInteraction | MessageInfoTypeClipped) => msg.messageType !== INFO_MESSAGE_CLIPPED) as Array<MessagePlanning | MessageInteraction>
-
-    const unArchivedMessages: Array<MessagePlanning | MessageInteraction> = nonTurnMessages.filter((message) => !message.details.archived)
-
```

Comment:
@lilit, is it worth doing `return e.message && e.message.Reference` - just in case there is some document without a `message` child?

Reply:
I think yes